language/stdlib/modules/LibraAccount.move

```diff
@@ -482,10 +482,9 @@ module LibraAccount {
         include Libra::AbortsIfNoCurrency<Token>;
     }
     spec schema DepositEnsures<Token> {
-        payer: address;
         payee: address;
         amount: u64;
-        ensures global<Balance<Token>>(payee).coin.value == old(global<Balance<Token>>(payee).coin.value) + amount;
+        ensures balance<Token>(payee) == old(balance<Token>(payee)) + amount;
```

Comment:
I didn't know that we can do this. It's apparent here, but just to make sure, can we omit "global" when accessing a resource in the global state?

Reply:
No we can't omit "global". `balance<Token>` is a move function defined in the module around line 1215. We can use `balance<Token>` in specs because it's pure.

Reply:
Ah, I was confused. Thanks for you answer.